### DISCUSSION TITLE
Unable to run .NET MAUI project in windows 11 pro virtual machine

### DISCUSSION THREAD

**Discussion by fingers10** [dotnet/maui#9487] [Q&A] [answered]
I'm trying to run my first `.net` `maui` project. I have my `Windows 11 pro` virtual machine running on `Windows 11 Home` host.

I have the pre-requisites configured.

1. Installed VS 2022 17.3.0.
2. Installed `.Net Multi-platform App UI Development`.

[![enter image description here][1]][1]

3. Enabled `Hyper-V` in windows features.

[![enter image description here][2]][2]

4. Run VS 2022 as Administrator.
5. Create `.Net Maui Default App`.
6. Run via Android `Pixel 5 - API 31 (Android 12.0 - API 31)` emulator.
7. Deploy failed Emulator cannot be started.

[![enter image description here][3]][3]

**Output:**
```
Build started...
Starting emulator pixel_5_-_api_31 ...
1>------ Build started: Project: MauiApp1, Configuration: Debug Any CPU ------
C:\Program Files (x86)\Android\android-sdk\emulator\emulator.EXE -netfast -avd pixel_5_-_api_31 -prop monodroid.avdname=pixel_5_-_api_31
Emulator pixel_5_-_api_31 cannot be started.
1>Skipping analyzers to speed up the build. You can execute 'Build' or 'Rebuild' command to run analyzers.
1>MauiApp1 -> C:\Users\Abdul Rahman\source\repos\MauiApp1\MauiApp1\bin\Debug\net6.0-android\MauiApp1.dll
2>------ Deploy started: Project: MauiApp1, Configuration: Debug Any CPU ------
2>Starting deployment to pixel_5_-_api_31 ...
2>Starting emulator pixel_5_-_api_31 ...
2>C:\Program Files (x86)\Android\android-sdk\emulator\emulator.EXE -netfast -avd pixel_5_-_api_31 -prop monodroid.avdname=pixel_5_-_api_31
2>Emulator pixel_5_-_api_31 cannot be started.
2>Deploy failed
========== Build: 1 succeeded, 0 failed, 0 up-to-date, 0 skipped ==========
========== Deploy: 0 succeeded, 1 failed, 0 skipped ==========
```

Please assist on what I'm missing.

  [1]: https://i.stack.imgur.com/wQHPx.png
  [2]: https://i.stack.imgur.com/3KHQj.png
  [3]: https://i.stack.imgur.com/TqHsI.png

**Comment by fingers10** [comment, marked as answer]
Here is how I solved using a physical android device.

Steps:
1. Enable `Developer Options` in your Mobile device
2. Enable `USB debugging` in your Mobile device

[![enter image description here][1]][1]

3. Connect your device to your virtual machine. A popup titled `Allow USB Debugging?` appers in phone. Allow it.

[![enter image description here][2]][2]

4. Open `MAUI` project
5. From the `Project run dropdown` select `Android Local Devices` and select your device.

[![enter image description here][3]][3]

6. Now run the app.
7. It will be deployed in your device and open in your device.
8. A preview will appear in `XAML Live Preview` window inside visual studio.

[![enter image description here][4]][4]

  [1]: https://i.stack.imgur.com/pKvRA.jpg
  [2]: https://i.stack.imgur.com/tU0J8.jpg
  [3]: https://i.stack.imgur.com/i2q5u.png
  [4]: https://i.stack.imgur.com/p2eaO.png

**Comment by jwy2020** [comment]
This is NOT an answer to the problem, but a work around.
